pingouin/tests/test_pairwise.py

```diff
@@ -495,8 +495,16 @@ def test_pairwise_corr(self):
         pairwise_corr(data, covar=['Age', 'Neuroticism'])
         with pytest.raises(AssertionError):
             pairwise_corr(data, covar=['Age', 'Gender'])
-        with pytest.raises(ValueError):
-            pairwise_corr(data, columns=['Neuroticism', 'Age'], covar='Age')
+        # with pytest.raises(ValueError):
+        #     pairwise_corr(data, columns=['Neuroticism', 'Age'], covar='Age')
```

Comment:
This test was meant to fail, but now non-partial `corr` is calculated instead, returning an empty list for covar. What do you think about this?

Reply:
The new behavior seems ok to me so ok to remove this test!